harvester/harvester-installer
# Move hostname to a dedicated page before network configuration to avoid crashes.

**Related issues:**
[#2937](https://github.com/harvester/harvester/issues/2937)
[#3112](https://github.com/harvester/harvester/issues/3112)

**Test plan:**
1. Prepare a small screen/low resolution environment, use ISO to boot into installer;
2. Go to `Hostname` page and set a hostname;
3. Go to `Network configuration` page and use a static IPv4 config to configure network;
4. Leave `MTU` empty, when focusing on `MTU` input box, press up arrow key;
5. See if installer crashed.

**Pictures:**
<img width="1212" alt="截屏2022-11-08 10 40 33" src="https://user-images.githubusercontent.com/9990804/200462089-b8524a27-15bc-4df9-ae6a-f17071e472d0.png">
<img width="1210" alt="截屏2022-11-08 10 41 01" src="https://user-images.githubusercontent.com/9990804/200462102-1d1a2f4a-e052-48da-b0f7-7544bcd5e313.png">
